### PR TITLE
Remove redundant babel plugins

### PR DESCRIPTION
**Pull request by nazar-pc**
Removal of those plugins makes it not polyfill the syntax leading to faster code

### CODE REVIEW

**Comment by jacogr** [comment]
100% - I have been eyeing these for a while since it does bloat and turns the code exceptionally ugly as well :)

Will keep these in the bin for now, when the build output gets changed "quite a bit" I believe it needs to go with a major version bump. Somebody using this somewhere is bound to break their bundler with this applied.

**Review by jacogr — APPROVED** [review]
Thank you. As mentioned, needs to go alongside a major version bum for wasm, utils & api.

**Comment by nazar-pc** [comment]
Turns out even in this case some stuff at least are still polyfilled (probably these plugins are now included by default), https://github.com/babel/babel/pull/13260 will make it possible to not polyfill private class properties in Node.js 12 (where they are supported natively).

**Comment by jacogr** [comment]
Still think it is a good idea to have less rather than more transformers explicitly. If you look at the outputs, they are horrendous to say the least.

**Comment by jacogr** [comment]
Additionally for the preset-env, you can use include to switch on/off stuff it would seem - https://babeljs.io/docs/en/babel-preset-env#options

TL;DR Lots of playing needed to clean the stuff up and get saner outputs :)

**Comment by jacogr** [comment]
Can I just ask to resolve the conflicts? (Sorry about those, had a bump in the middle)

**Comment by nazar-pc** [comment]
Done

**Comment by jacogr** [comment]
The private `#instanceId` stuff will be problematic - not all browsers support it atm, so that probably needs to make the way back.

**Comment by nazar-pc** [comment]
I was looking at https://caniuse.com/mdn-javascript_classes_private_class_fields and don't see anything modern that doesn't support it

**Comment by jacogr** [comment]
The issue is FF - The ESR version is 78, so that is pre-support. (Sadly I'm getting a lot of support queries on that browser atm)

**Comment by nazar-pc** [comment]
78 ESR is EOL since 2 days ago, I see no problems dropping it at this point, but it might require bumping major versions so people can stick to outdated Polkadot.js version if they need support for dead browser.

**Comment by jacogr** [comment]
Will merge, however going to revert the private changes for now - it is never fun when the world breaks :)

**Review by jacogr — APPROVED** [review]
